src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/FindAllReferencesHandler.cs

```diff
@@ -81,14 +92,18 @@ internal async override Task<VSReferenceItem[]> HandleRequestAsync(ReferencePara
                 throw new ArgumentNullException(nameof(clientCapabilities));
             }
 
+            _logger.LogInformation($"Starting request for {request.TextDocument.Uri}.");
```

Comment:
For a lot of these "starting X" I wonder if it'd be better to LogTrace since they're effectively scoped, duplicate information to the original StreamJsonRpc request. I think in general we may want to "LogTrace" for all of the messages that happen "all the time" and LogInformation for some of the less common scenarios (like provisional completions). Thoughts?

Reply:
Oh, if a scoped object is available for Start/End comments I 100% agree, didn't realize this log API had those.

Reply:
Created https://github.com/dotnet/aspnetcore/issues/30408 to help address this concern. Want to avoid doing it here as this PR is getting a bit bloated.